src/components/TodoList.tsx

```diff
@@ -0,0 +1,14 @@
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+type TodoListProps = {
+  items: Todo[]
+};
+
+export const TodoList = ({ items }: TodoListProps) => {
+  return (
+    <ul className="todo-list" data-cy="todosList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings

Reply:
I use `ul`, `li`. I haven't padding and dots. Can You describe this comment @loralevitska?